test/Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj

```diff
@@ -58,8 +58,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Reflection\Dependencies\" />
-    <Folder Include="BCLFeatures\" />
-    <Folder Include="BCLFeatures\ETW\" />
```

Comment:
I assume the change also deletes the files and folders (GitHub doesn't show this in the PR I guess).